packages/components/src/internal/components/entities/EntityLineageEditModal.tsx

```diff
@@ -0,0 +1,197 @@
+    );
+
+    if (!queryModel || !nonAliquots) {
+        return null;
+    }
+
+    const numNonAliquots = Object.keys(nonAliquots).length;
+
+    if (numNonAliquots === 0) {
+        return (
+            <Modal show onHide={onCancel}>
+                <Modal.Header closeButton>
+                    <Modal.Title>Cannot Edit {parentNounPlural}</Modal.Title>
+                </Modal.Header>
+
+                <Modal.Body>
+                    <div>The {lcParentNounPlural} for aliquots cannot be changed.</div>
+                </Modal.Body>
+
+                <Modal.Footer>
+                    <Button bsClass={'btn btn-default'} onClick={onCancel} >
+                        Dismiss
+                    </Button>
+                </Modal.Footer>
+            </Modal>
+        )
+    }
+
+    return (
+        <Modal bsSize="large" show onHide={onCancel}>
+            <Modal.Header closeButton>
+                <Modal.Title>Edit {parentNounPlural} for {numNonAliquots} Selected {capitalizeFirstChar(getEntityNoun(childEntityDataType, numNonAliquots))}</Modal.Title>
+            </Modal.Header>
+
+            <Modal.Body>
+                <div className={"bottom-spacing"}>
+                    <p>
+                        Values provided here will <b>replace</b> the existing {lcParentNounPlural} of the chosen types for the selected {getEntityNoun(childEntityDataType, nonAliquots.length)}.
+                        Remove {parentNounSingular} Types from the form below that you do not wish to change the value of.
+                    </p>
+                    <p>
+                        To see details of the existing {lcParentNounPlural}, choose "Cancel" here then "Edit Selected {capitalizeFirstChar(childEntityDataType.nounPlural)} in Grid" from the "Manage" menu.
+                    </p>
+                </div>
+                {(numAliquots > 0 && !submitting) && <Alert bsStyle={'info'}> {Utils.pluralize(numAliquots, 'aliquot was', 'aliquots were')} among the selections. Lineage for aliquots cannot be changed.</Alert>}
+                <Alert bsStyle={'danger'}>{errorMessage}</Alert>
+
+                <Progress
+                    modal={false}
+                    estimate={numNonAliquots  * 10}
+                    toggle={submitting}
+                />
+                {!submitting &&
+                    <ParentEntityEditPanel
+                        auditBehavior={auditBehavior}
+                        canUpdate={true}
+                        childQueryInfo={queryModel.queryInfo}
+                        childData={undefined}
+                        parentDataTypes={parentEntityDataTypes}
+                        childName={undefined}
+                        childNounSingular={childEntityDataType.nounSingular}
+                        key={`parent${parentNounPlural}-${queryModel.id}`}
+                        onUpdate={onConfirm}
+                        title="Details"
```

Comment:
minor: should the title prop for this component be optional or have a default value? seems a little odd to supply this and then say `includePanelHeader={false}`

Reply:
Good thought.